tests.py

```diff
@@ -226,14 +226,10 @@ def test_noise_sfts(self):
         add_signal_writer = self.writer_class_to_test(
             label="test_noiseSFTs_add_signal",
             outdir=self.outdir,
-            duration=None,
-            Tsft=self.Tsft,
-            tstart=None,
             SFTWindowType=self.SFTWindowType,
             SFTWindowBeta=self.SFTWindowBeta,
-            sqrtSX=0,
```

Comment:
Seems to make sense to remove these, with `noiseSFTs` I hope they're ignored anyway.

Reply:
They are. The explanation is "This test precedes the overhaul of `noiseSFTs`".